CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 This document outlines the changes from version to version.
 
+## 2.3.0
+
+- Added `Array<Result>.collect()` and `Array<Maybe>.collect()`
```

Comment:
Just need to update this again

Reply:
794f605affeecd16590bed47f1b250d4ac91d860